### PR TITLE
[Enhancement] #356 dropdown exempt or non exempt employee/job

### LINKED ISSUE #356: [Enhancement] make a dropdown to choose exempt or non-exempt
employee/job

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #356

### CODE REVIEW

File: app/templates/account/employee/index/edit/job.hbs [hunk 1]
```diff
@@ -16,8 +16,19 @@
 <div class="three fields">
   <div class="field">
     <label for="exemption-status">Exemption Status</label>
-    Non-exempt
-    {{ui-checkbox class="toggle" label="Exempt" checked=model.exempt onChange=(action (mut model.exempt))}}
+    {{#ui-dropdown class="selection" selected=model.exempt onChange=(action (mut model.exempt)) as |execute mapper|}}
+      <div class="default text" value=model.exempt></div>
+      {{log model.exempt }}
```
Comment (James1x0) [inline app/templates/account/employee/index/edit/job.hbs]: Remove log

File: app/templates/components/search/result-item.hbs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{{#if model}}
+  <div class="search-result-item__title">
+    {{log model}}
```
Comment (James1x0) [inline app/templates/components/search/result-item.hbs]: 👎

File: app/templates/components/search/result-item.hbs [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    {{display/employee-picture url=employee.picture employeeId=employee.id}}
+    {{model.fullName}}
+  </div>
+  <div class="search-result-item__description">
+    {{log 'hey' model}}
```
Comment (James1x0) [inline app/templates/components/search/result-item.hbs]: 👎 Hey.

File: tests/integration/components/search/result-item-test.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import { module, test } from 'qunit';
```
Comment (James1x0) [inline tests/integration/components/search/result-item-test.js]: ??? this file aint yo file

**Comment by James1x0** [comment]
Need a screenshot too

**Comment by JavaLavaMT** [comment]
image

**Comment by James1x0** [comment]
@JavaLavaMT Change label "Non Exempt" to "Non-exempt"

**Comment by JavaLavaMT** [comment]
image

**Comment by James1x0** [comment]
@JavaLavaMT Resolve conflicts plz

**Pull request merged**